core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -168,4 +169,17 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         getResult.writeTo(out);
     }
+
+	@Override
+	public String toString() {
+		try {
```

Comment:
@nik9000 Are you suggesting not implementing `SearchResponse.toString()` but using `Strings.toString()` in application code, or implementing `SearchResponse.toString()` via using `Strings.toString()`?